examples/npm/discord/commands/run.js

```
@@ -0,0 +1,45 @@
+      filter: x => x.customId === 'run-modal' && x.user.id === i.user.id,
+    }).catch(() => null);
+    if (!submitted) return;
+
+    const code = submitted.fields.getTextInputValue('code');
+
+    let buf = '';
+    const sink = { write(s) { buf += s; return true; } };
+    const captured = new console.Console({ stdout: sink, stderr: sink });
+
+    try {
+      await eval(`(async (console) => { ${code} })`)(captured);
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)